src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -30,6 +30,10 @@ HARDWARE_INTRINSIC(Sve,           Compute16BitAddresses,
 HARDWARE_INTRINSIC(Sve,           Compute32BitAddresses,                                            -1,      2,      true,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_adr,        INS_invalid,        INS_sve_adr,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen)
 HARDWARE_INTRINSIC(Sve,           Compute64BitAddresses,                                            -1,      2,      true,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_adr,        INS_invalid,        INS_sve_adr,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen)
 HARDWARE_INTRINSIC(Sve,           Compute8BitAddresses,                                             -1,      2,      true,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_adr,        INS_invalid,        INS_sve_adr,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen)
+HARDWARE_INTRINSIC(Sve,           ConditionalExtractAfterLastActiveElement,                         -1,      3,      true,  {INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta},  HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_HasScalarInputVariant|HW_Flag_SpecialCodeGen)
+HARDWARE_INTRINSIC(Sve,           ConditionalExtractAfterLastActiveElementAndReplicate,             -1,      3,      true,  {INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta,     INS_sve_clasta},  HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_HasScalarInputVariant)
```

Comment:
`*Replicate` APIs do not have scalar variant and is supposed to generate [CLASTA (vectors)](https://docsmirror.github.io/A64/2023-06/clasta_z_p_zz.html) instruction. It doesn't need `HW_Flag_HasScalarInputVariant` flag.

Reply:
Done.